recipes/scalene/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "scalene" %}
+{% set version = "1.5.10" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # See https://pypi.org/project/scalene/#files
+  sha256: 8e2e2ae14acdca56d5bbf33e158814702410f0a72d4e690fce837b811ffadbfc
+
+build:
+  skip: true  # [py<38]
+  skip: true  # [win]
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - python
+    - pip
+    - setuptools_scm
+    - wheel
+  run:
+    - python
+    - rich >=9.2.0
+    - cloudpickle >=1.5.0
+    - numpy
```

Comment:
```suggestion
```
I think you're not understanding my point. `numpy` is not listed in the package `install_requires`. Your conda recipe should not list `numpy` as a dependency if `numpy` is not imported (which it isn't; except in `scalene.old`).

Reply:
Gotcha. Outside of this file, it looks like [there are a few uses]( https://github.com/plasma-umass/scalene/search?p=2&q=numpy ), but mainly tests. Not sure how [this profile]( https://github.com/plasma-umass/scalene/blob/56719723b694030905f25223dd207c6a97909e0f/scalene/scalene-gui/profile.json#L80 ) figures in.